internal/payloads/instance_types_payload.go

```diff
@@ -19,9 +19,9 @@ func (s *InstanceTypeResponse) Render(_ http.ResponseWriter, _ *http.Request) er
 	return nil
 }
 
-func NewListInstanceTypeResponse(sl *[]clients.InstanceType) []render.Renderer {
-	list := make([]render.Renderer, 0, len(*sl))
-	for _, instanceType := range *sl {
+func NewListInstanceTypeResponse(sl []*clients.InstanceType) []render.Renderer {
+	list := make([]render.Renderer, 0, len(sl))
+	for _, instanceType := range sl {
 		list = append(list, &InstanceTypeResponse{instanceType})
 	}
```

Comment:
This change makes sure we are not copying instance types over and over again, it is now slice of pointers, not pointer to slice. Slices are effectively passed as reference.